src/main/java/org/jenkinsci/plugins/authorizeproject/strategy/SpecificUsersAuthorizationStrategy.java

```diff
@@ -73,25 +77,76 @@ public String getUserid() {
         return userid;
     }
     
-    private final boolean noNeedReauthentication;

```

Comment:
I plan to introduce an alternate feature like "Don't restrict users to configuring the job":

    * `hasConfigurePermission` returns always `true` if checked.
        * `readResolve` blocks CLI and RestWebAPI by a user other than the one to run as.
    * This is an option expected to be used when appropriate permissions are configured for that job.

I'll redefine `noNeedReauthentication` with `transitive` for migration from old configurations.